packages/genesys-spark-components/src/components/stable/gux-form-field/gux-form-field.service.ts

```diff
@@ -83,6 +83,11 @@ export function validateFormIds(
     error.setAttribute('id', errorId);
     describedByIds.push(errorId);
```

Comment:
The error text is now connected to the input twice. `aria-describedby` and `aria-errormessage`. Should we remove the error id from the  `aria-describedby`?

Reply:
Removing based on Jason's comment above, the aria-describedby hack is GEFN.